gateway-release-common/home/bin/knox-functions.sh

```diff
@@ -50,6 +50,17 @@ DEFAULT_APP_STATUS_TEST_RETRY_SLEEP=2
 ##### common functions #####
 ############################
 
+function setVerbose() {
```

Comment:
nitpick: May be accommodate shortform as well `-v`

Reply:
KnoxCLI only supports `--verbose`, hence the long version only. Maybe later...

Reply:
i see makes sense :)